src/components/Tree/Tree.stories.tsx

```diff
@@ -0,0 +1,90 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { ReactComponent as IconGuidelines } from "@components/Icon/Svg/Guidelines.svg";
+import { ReactComponent as IconDocument } from "@components/Icon/Svg/Document.svg";
+import { Meta, Story } from "@storybook/react";
+import { TreeNode } from "./Node";
+import Tree, { TreeProps } from "./Tree";
+import { IconSize } from "@components/Icon/Icon";
+
+export default {
+    title: "Tree",
+    component: Tree,
+} as Meta;
+
+const TreeTemplate: Story<TreeProps> = (args) => <Tree {...args} />;
+
+export const Default = TreeTemplate.bind({});
+
+Default.args = {
```

Comment:
how about adding the other props? like `onSelect` and `selectedNodeId`

Reply:
e4ccece14380b7a3602047727defd88344e03feb

onSelect shows the selected id in storybook